test/forms/sections/items/ItemPackagingShippingMarksFormProviderSpec.scala

```diff
@@ -49,5 +75,23 @@ class ItemPackagingShippingMarksFormProviderSpec extends StringFieldBehaviours {
       fieldName,
       requiredError = FormError(fieldName, requiredKey)
     )
+
+    Seq(ItemPackagingShippingMarksMessages.English) foreach { messagesForLanguage =>
+      "display an error when the shipping mark value has been used elsewhere" in {
+        form.bind(Map(fieldName -> "mark 2")).errors must contain(FormError(fieldName, errorShippingMarkNotUnique(testPackagingIndex2)))
+      }
+
+      "not display an error when the shipping mark value is the same as the one we are modifying" in {
+        form.bind(Map(fieldName -> "mark 5")).hasErrors mustBe false
+      }
+
+      "not display an error when the shipping mark value is changed to something not used before" in {
+        form.bind(Map(fieldName -> "mark 5555555555555555")).hasErrors mustBe false
+      }
```

Comment:
Could there be a test for when there are no shipping marks that exist?

Reply:
Added https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/306/commits/fab120bcc4858b2ad14d4b4456ba6b96ae6e951d